core/chain-configs/src/genesis_config.rs

```diff
@@ -194,7 +194,8 @@ pub struct GenesisConfig {
     pub minimum_stake_divisor: u64,
     /// Layout information regarding how to split accounts to shards
     #[serde(default = "default_shard_layout")]
-    #[default(ShardLayout::v0_single_shard())]
+    // FIXME eagr what should be the default?
+    #[default(ShardLayout::v0(1, 0))]
```

Comment:
Ideally it should use the `single_shard` method that returns the most recent (today it's V2) shard layout. 

nit: The convention here seems to be to use the `default_` function to provide the default value.